corehq/apps/motech/templates/motech/motech_openmrs_concepts.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'style/base_section.html' %}
+{% load hq_shared_tags %}
+
+{% block js %}
+    {{ block.super }}
+    <script src="{% static 'motech/js/motech_openmrs_concepts.js' %}"></script>
+{% endblock %}
+{% block page_content %}
+    {% registerurl 'all_openmrs_concepts' domain %}
+    {% registerurl 'openmrs_sync_concepts' domain %}
+    <div id="openmrs_concepts_template" class="ko-template">
+        <button class="btn btn-default" data-bind="
+            disable: syncButton.inProgress,
+            click: syncButton.sync
+        ">
+            <!--ko ifnot: syncButton.inProgress-->Sync from OpenMRS<!--/ko-->
```

Comment:
That's fair. I was in kind of "this isn't really going to be live yet so it doesn't matter" mode, but you're right—it's way easier to do this kind of thing as you go along than it is to do them in bulk after the fact.

- [ ] Translate all easily-translatable strings in this PR's diff
- [ ] Add a todo comment next to any hard-to-translate string that should eventually be translated (if any).